forge-gui-mobile/src/forge/adventure/scene/DuelScene.java

```diff
@@ -202,10 +210,17 @@ public void setDungeonEffect(EffectData E) {
     public void enter() {
         GameHUD.getInstance().unloadAudio();
         Set<GameType> appliedVariants = new HashSet<>();
-        appliedVariants.add(GameType.Constructed);
+        if (eventData!= null && eventData.eventRules != null){
+            appliedVariants.add(eventData.eventRules.gameType);
+        }
+        else{
+            appliedVariants.add(GameType.Adventure);
```

Comment:
the status scene should be updated and the achievements for the adventure gametype - achievementcollection

Reply:
Good point. Will implement & test these this afternoon.